lib/useBackHandler.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from 'react'
+import { BackHandler } from 'react-native'
+
+export default () => {
+  const [press, setPress] = useState(false)
+
+  backPressed = () => {
+    this.props.navigation.goBack();
+    setPress(true);
+  }
+
+  useEffect(() => {
+    BackHandler.addEventListener('hardwareBackPress', this.backPressed);
```

Comment:
you don't need ``this ``. Just use backPressed

Reply:
both work fine, but i selected the way this library is written so that contributors won't confuse.